src/components/MapImageSlider.vue

```diff
@@ -1770,10 +1770,10 @@
         <text
           id="title-svg"
           transform="translate(0 0)"
-          style="font-size: 14px; font-weight: 600; fill: rgb(139, 139, 139);"
+          style="font-size: 10px; font-style: italic; fill: rgb(139, 139, 139);"
         >State-level trends in USGS streamgaging<tspan
           x="0"
-          y="15"
+          y="10"
         >1890 - present</tspan></text>
```

Comment:
I like the idea of this being more of a sentence than a title. What do you think? We could just make line 2, "from 1890 to present" instead?

Reply:
I think having all the styling in R code OR in the CSS is fine but I think we may get confused if it is in both places (that is what the culprit of the arrow not being hidden after mouseover was from)